src/logonserver/Server/Master.cpp

```diff
@@ -142,9 +142,6 @@ void LogonServer::Run(int argc, char** argv)
     std::string host = Config.MainConfig.GetStringDefault("Listen", "Host", "0.0.0.0");
     std::string shost = Config.MainConfig.GetStringDefault("Listen", "ISHost", host.c_str());
 
-    clientMinBuild = 12340;
-    clientMaxBuild = 12340;
```

Comment:
A config is useless... there is nothing the user is allowed to configure. We decide which versions are supported based on the programm.